### PR TITLE
Keyword: Saddle

### LINKED ISSUE #4803: Keyword: Saddle
* new Mount Creature Type
* Saddle N means similar to Crew N to "tap creatures with total power N or more", Saddle only as Sourcery
* the Creature becomes saddled (new Card Property) until EOT
* other Effects might check for being saddled

### PR DESCRIPTION
**Pull request by Hanmac**
Closes #4803

### CODE REVIEW

**Review by tool4ever — DISMISSED** [review]
should be good besides AI

**Comment by Agetian** [comment]
Looks good to me! Can we hook this up to the Crew AI and make distinctions where necessary, perhaps, or do you think this needs its own AI path?

**Comment by Hanmac** [comment]
@Agetian probably needs different AI path, because it goes over AlterAttribute and not Animate

File: forge-game/src/main/java/forge/game/card/CardUtil.java [hunk 1]
```diff
@@ -268,6 +268,7 @@ public static Card getLKICopy(final Card in, Map<Integer, Card> cachedMap) {
         newCopy.setMonstrous(in.isMonstrous());
         newCopy.setRenowned(in.isRenowned());
         newCopy.setSolved(in.isSolved());
+        newCopy.setSaddled(in.isSaddled());
```
Comment (tehdiplomat) [inline forge-game/src/main/java/forge/game/card/CardUtil.java:271]: Once you rebase, this has moved to the CopyCardService

**Comment by Hanmac** [comment]
@Agetian @tool4ever i tried to add some AI logic for Suspected and Saddled

**Review by Agetian — APPROVED** [review]

**Comment by Agetian** [comment]
Looks good for the initial implementation imo, we can flesh out the AI details over time :)

**Pull request merged**